src/watchful/attributes.py

```diff
@@ -1,6 +1,7 @@
 """
 This script provides the functions required for data enrichment.
 """
+
```

Comment:
Yeah, I'd be curious what formatting changed this, but I'd also like to see these reverted. Formatting-only changes to files should not live in patches that have actual functionality changes.

Reply:
see above

Reply:
done